lib/routers/establishment/project-versions.js

```diff
@@ -60,6 +76,20 @@ const validateAction = (req, res, next) => {
   return next(new BadRequestError());
 };
 
+router.get('/question/:question', (req, res, next) => {
+  const { ProjectVersion } = req.models;
+  Promise.resolve()
+    .then(() => ProjectVersion.getAll(req.project.id))
+    .then(versions => {
+      res.response = versions.map(v => {
+        let node = getNode(v.data, req.params.question);
+        return {versionId: v.id, value: node};
+      });
+      next();
+    })
+    .catch(next);
```

Comment:
It feels like this is doing too much by returning _all_ versions. 

We have code elsewhere that loads the previous and granted versions of a PPL in order to do comparison between them, and it feels to me like we should be using that same code to get the versions here rather than loading every single version.